report/src/system_evaluation/phase_one_evalation.tex

```diff
@@ -0,0 +1,23 @@
+\section{Phase One: Map Creation}\label{sec:eval_phase1}
+To collect measurements for the creation of the map, a person holding a Samsung Galaxy A53 5G phone, with the application described in Section \ref{sec:data_collection_implementation} installed and running, traverses systematically through the room and stops at each collection point to collect measurements for the map. 
+We will refer to this person as the \textit{collector}.
+
+During the creation of the map, the collector is responsible for correctly labelling the measurements collected at each collection point shown in Figure \ref{fig:room_partition_measurements}. 
+The collector is also responsible for holding the phone in roughly the same heigt when creating the map. 
+During this experiment, the collector had the phone out in front of their face when collecting measurements for the map. 
+During the map creation, the collector collects measurements for all four beacons until the a standard deviation of under 6.5 \todo{insert unit of standard deviation. What does 6.5 mean? AZ} and a minimum of $40$ measurements are established.
+This is done by selecting all beacons in the developed application and using the functionality described in Section \ref{sec:first_phase}.
+
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=\textwidth]{images/experiment_map_creation.png}
+    \caption{A sketch of the meeting room partitioned into a grid decorated with beacon placements (red antenna) and measurement locations (circles with letters). The figure how the measurements are collected systematically in the meeting room.}
+    \label{fig:experiment_map_creation}
+\end{figure}
+Whenever the collector stops at one of the collection points depicted in Figure \ref{fig:room_partition_measurements}, they press a button in the app, which starts measuring after five seconds, as depicted in Figure \ref{fig:collecting_data}.
+The phone provides auditory feedback when it has collected enough measurements (in this case 40) and has reached a low enough standard deviation (6.5).
```

Comment:
I don't know if stable is the right word. We can have many measurements that consistently gives a really high standard deviation. This would be stable. The results of these measurements would still not be useful, since they deviate a lot.
Don't we wait until we have a low standard deviation?

Reply:
Mostly don't care so long as the parentheses re removed.

```suggestion
The phone provides auditory feedback when it has collected enough measurements and has reached a low enough standard deviation.
```